openstack/02b-MAChigh/schedule.c

```diff
@@ -521,6 +529,49 @@ slotOffset_t schedule_getNextActiveSlotOffset() {
 }
 
 /**
+\brief return slotOffset of closest active slot

```

Comment:
This is called when receiving the first EB by activity_synchronize_endOfFrame. It return the closest future active slot.